config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "customers/:id", to: "customers#show"
```

Comment:
To complete the form functionality, we'd want an additional route. It's creating a  joins record so it shouldn't go back to the show route directly, but we can eventually redirect it to there after the joins record creation is finished. 

Consider: `post "/customers/:customer_id/items", to: "customer_items#create"`
We can direct the form to this controller action - it's in a separate controller to indicate it's a joining of these 2 resources (if we did it in the CustomersController `create` action, we wouldn't then be able to use it down the road to create new customers, therefore it's not as RESTful/extendable. )

In this `create` action, we could create the joins record based off the customer ID in the path and the item ID entered in the form. Then, you can redirect back to  "customers/:id" to get back to the show page